src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -2623,7 +2623,7 @@ public void Bug1095618(CSharpParseOptions parseOptions)
         [MemberData(nameof(AllProviderParseOptions))]
         public void ConsistentErrorMessageWhenProvidingNullKeyFile(CSharpParseOptions parseOptions)
         {
-            var options = new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary, cryptoKeyFile: null);
```

Comment:
The default value for cryptoKeyFile is null, so I didn't add `.WithCryptoKeyFile(null)` explicitly as it's redundant.

If you think the test is more clearer with explicitly setting WithCryptoKeyFile, let me know to update this.

Reply:
Consider adding an `Assert.Null(options.CryptoKeyFile)` to clearly convey this requirement